databus-util-cmdline/databus-util-cmdline-impl/src/main/java/com/linkedin/databus/util/FieldToAvro.java

```diff
@@ -202,7 +202,7 @@ else if (typeInfo instanceof CollectionTypeInfo)
     field.put("default", null);
 
     // Field type
-    String[] type = new String[] { "null", typeInfo.getPrimitiveType().getAvroType() };
+    String[] type = new String[] { typeInfo.getPrimitiveType().getAvroType(), "null"};
```

Comment:
This is a significant change. Putting the "null" type second means that "null" cannot be the default value anymore.

Why is this necessary.